src/test/audio_label_test.py

```diff
@@ -45,6 +48,21 @@ def test_length(self):
         label = AudioLabel(1.23,99.35,"text")
 
         self.assertAlmostEqual(label.length(),98.12)
+
+    def test_assign_audio(self):
+        in_point = 1.345
+        out_point = 2.345
+        text = 'here is the text'
+
+        label = AudioLabel(in_point=in_point, out_point=out_point, text=text,)
```

Comment:
Just curious about the trailing comma in the function parameters, is that a python thing?  or just a typo?

Reply:
Good point! We haven't put in a formatter yet. We should do that now.

Reply:
The formatter has taken care of this.